src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/CultureInfoAdapter.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace System.Runtime.Serialization
+{
+    [DataContract(Name = "CultureInfo", Namespace = "http://schemas.datacontract.org/2004/07/System.Globalization")]
+    public struct CultureInfoAdapter
+    {
+        private string _name;
+        private int _dataItem;
+        private bool _isReadOnly;
+        private bool _useUserOverride;
+
+        [DataMember(Name = "calendar", IsRequired = true)]
+        public string Calendar { get; set; }
+
+        [DataMember(Name = "compareInfo", IsRequired = true)]
+        public string CompareInfo { get; set; }
+
+        [DataMember(Name = "cultureID", IsRequired = true)]
+        public int CultureID { get; set; }
+
+        [DataMember(Name = "dateTimeInfo", IsRequired = true)]
+        public string DateTimeInfo { get; set; }
+
+        [DataMember(Name = "m_dataItem", IsRequired = true)]
+        public int DataItem
+        {
+            get { return _dataItem; }
+            set { _dataItem = value; }
```

Comment:
Why are some properties manually getting/setting fields, while others are auto-implemented? Could all the properties be auto-implemented properties?